src/Execution/ValuesResolver.php

```diff
@@ -558,36 +626,38 @@ protected function printPath(?Path $path)
      * @return mixed
      * @throws ExecutionException
      */
-    protected function coerceValueForVariableNode(
-        VariableNode $variableNode,
-        TypeInterface $argumentType,
-        string $argumentName,
-        array $variableValues,
-        $defaultValue
-    ) {
-        $variableName = $variableNode->getNameValue();
-
-        if (!empty($variableValues) && isset($variableValues[$variableName])) {
-            // Note: this does not check that this variable value is correct.
-            // This assumes that this query has been validated and the variable
-            // usage here is of the correct type.
-            return $variableValues[$variableName];
-        }
-
-        if (null !== $defaultValue) {
-            return $defaultValue;
-        }
-
-        if ($argumentType instanceof NonNullType) {
-            throw new ExecutionException(
-                \sprintf(
-                    'Argument "%s" of required type "%s" was provided the variable "$%s" which was not provided a runtime value.',
-                    $argumentName,
-                    $argumentType,
-                    $variableName
-                ),
-                [$variableNode]
-            );
-        }
-    }
+//    protected static function coerceValueForVariableNode(
```

Comment:
Can you remove this instead of commenting it out?

Reply:
Sure. I apparently forgot to remove the unused method.